apps/librelingo_json_export/tests/test_challenges_pairing_challenge.py

```diff
@@ -0,0 +1,54 @@
+import collections
+
+import pytest
+from librelingo_fakes import fakes
+from librelingo_json_export.challenge_types import get_pairing_challenge
+
+Example = collections.namedtuple(
+    "Example", ["word", "course", "expected_result", "name"]
+)
+
+
+@pytest.mark.parametrize(
+    "example",
+    [
+        Example(
+            fakes.pairs1,
+            fakes.course1,
+            {
+                "id": "33e2a4a86ca9",
+                "type": "pairing",
+                "pairs" : [
+                    {"meaningInSourceLanguage": "dog", "formInTargetLanguage": "perro"},
+                    {"meaningInSourceLanguage": "cat", "formInTargetLanguage": "gata"}
+                ],
+                "priority": 0,
+                "group": "b4be982a043f",
+            },
+            "data_set_1",
+        ),
+        Example(
+            fakes.pairs2,
+            fakes.course1,
+            {
+                "id": "33e2a4a86ca9",
+                "type": "pairing",
+                "pairs" : [
+                    {"meaningInSourceLanguage": "dog", "formInTargetLanguage": "perro"},
+                    {"meaningInSourceLanguage": "cat", "formInTargetLanguage": "gata"},
+                    {"meaningInSourceLanguage": "duck", "formInTargetLanguage": "pato"},
+                    {"meaningInSourceLanguage": "lion", "formInTargetLanguage": "leon"},
+                    {"meaningInSourceLanguage": "horse", "formInTargetLanguage": "caballo"}
+                ],
+                "priority": 0,
+                "group": "b4be982a043f",
+            },
+            "data_set_2",
+        ),
+    ],
+    ids=lambda example: example.name,
+)
+def test_get_pairing_challenge(example):
+    print(example.word)
```

Comment:
```suggestion
```

it should not be necessary to print this here, instead we should rely on pytest in order to provide output that's easy to debug. I think that should already be the case, right? I mean if you run pytest, it should tell you what values `get_pairing_challenge` was called with, in case the test fails

Reply:
Oh yes, my mistake. It was just for debugging purposes and it looks like I forgot to remove it.